torch/functional.py

```diff
@@ -386,16 +390,16 @@ def meshgrid(*tensors):
             True
 
         """
-        return _meshgrid(*tensors)
+        return _meshgrid(*tensors, indexing=indexing)
```

Comment:
If this is going to warn people to pass `indexing`, which seems very reasonable, then this should probably update the above example to always including the `indexing` kwarg and demonstrate our recommended practice

Reply:
fyi not all the meshgrid uses in the above examples were updated to use "indexing"

Reply:
True! That is intentional. I don't use it in the one where we plot the output. Most of the time in a plot you will want "xy" indexing, and eventually that becomes the default. I don't want to have a temporary example where we have `indexing='ij'`, since we don't really want anyone copying that.

I think when we add the option (i.e. next PR) to have `indexing='xy'` it makes sense to update that example.